packages/react-components/src/stories/Introduction.mdx

```diff
@@ -15,9 +15,14 @@ export const link =
 
 <Meta title="Introduction" />
 
-# B.C. Design System React Components
+# B.C. Design System React Components v{packageJson.version}
 
-This is a UI workshop for React components included in the [B.C. Design System](https://gov.bc.ca/designsystem). This app provides:
+<Subtitle>
```

Comment:
Looks good!

<img width="1840" alt="Storybook introduction page" src="https://github.com/user-attachments/assets/6744c088-a247-4791-899d-858279847c41">

Reply:
Fixed in [993b7e0](https://github.com/bcgov/design-system/pull/419/commits/993b7e0e7094d5ee9b3953408d80c4db85c5c09f)!